File: server/models/Customer.py


from sqlalchemy import Column, Integer, String
from sqlalchemy.ext.declarative import declarative_base
from config import Base


class Customer(Base):
    __tablename__ = 'customers'
    custId = Column(Integer, primary_key=True)
    cityId = Column(Integer)
    lastName = Column(String)
    firstName = Column(String)
    phone = Column(String)
    email = Column(String)
    pwd = Column(String)

File: client-end/src/app/models/Customer.ts
export class Customer{
    custId:number=0
    cityId:number=0
    lastName:string = ''
    firstName:string = ''
    phon:string=''
    email:string=''
    pwd:string=''


    constructor( custId:number,cityId:number ,lastName:string,
        firstName:string,  phon:string ,email:string, pwd:string       ){
            this.custId=custId
            this.cityId=cityId
            this.lastName=lastName
            this.firstName=firstName
            this.phon=phon
            this.email=email
            this.pwd=pwd
}

   

}


